core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -813,14 +813,14 @@ private[spark] class SparkSubmit extends Logging {
       mainClass = Utils.classForName(childMainClass)
     } catch {
       case e: ClassNotFoundException =>
-        logWarning(s"Failed to load $childMainClass.", e)
+        logError(s"Failed to load class $childMainClass.")
```

Comment:
Here I tried append `e.getMessage` to the output, but it seems the error message equals  the class name `childMainClass`. e.g.
```
Error: Failed to load class foo: foo.
```
So I think we can output without the exception message.